### ISSUE TITLE
MK3 3.2.0-RC1 - automatic temp calibration not consistent

### ISSUE THREAD

**Issue by Crunch69** [prusa3d/Prusa-Firmware#640]
I ran several times and offsets where produced and saved issue is they are not consistent and vary each and every time example

PINDA temperature: 35 (mm):0
PINDA temperature: 40.28 (mm):0.05
PINDA temperature: 45.29 (mm):0.04
PINDA temperature: 49.87 (mm):0.06
PINDA temperature: 55.16 (mm):0.10
PINDA temperature: 60.16 (mm):0.16

PINDA temperature: 35 (mm):0
PINDA temperature: 39.98 (mm):0.06
PINDA temperature: 45.07 (mm):0.08
PINDA temperature: 49.97 (mm):0.12
PINDA temperature: 55.22 (mm):0.16
PINDA temperature: 60.19 (mm):0.24

**Comment by stahlfabrik** [comment]
Hi Crunch69.

Have a look at https://shop.prusa3d.com/forum/hardware-firmware-and-software-help-f64/1st-layer-problems-in-depth-look-at-software-pinda-t14815.html

TLDR:
You could run automatic temp calibration many times and note down the results. Then calculate mean values and pump those to eeprom using M861 Sx Iy.

For me - the automatic calibration does not yield good results.

Thus I propose a manual print based temperature calibration as described in my forum post above. You will get perfect results with this method.
